agent/internal/task_config.go

```diff
@@ -19,14 +19,25 @@ import (
 )
 
 type TaskConfig struct {
-	Distro             *apimodels.DistroView
-	ProjectRef         model.ProjectRef
-	Project            model.Project
-	Task               task.Task
-	BuildVariant       model.BuildVariant
-	Expansions         util.Expansions
-	NewExpansions      *agentutil.DynamicExpansions
-	DynamicExpansions  util.Expansions
+	Distro       *apimodels.DistroView
+	ProjectRef   model.ProjectRef
+	Project      model.Project
+	Task         task.Task
+	BuildVariant model.BuildVariant
+
+	// Expansions store the fundamental expansions set by Evergreen.
```

Comment:
This didn't change code, only comments. I wanted to add some because everytime I go back to the TaskConfig, I question why there are three Expansion related things.

I also noticed we could maybe do away with DynamicExpansions, but I'm not sure